### DISCUSSION TITLE
3인칭 뷰에서 보이는 시점을 변경했을때 다시 원래 위치로 돌아가는 이슈

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#6] [제보하기]
내가 원하는 뷰를 위해서 시점을 돌렸는데,  마우스 클릭을 떼는 순간 원래 위치로 조금씩 돌아갑니다.
고정이 되면 좋을 것 같습니다. 

시점이 원래 위치로 돌아가고 있는 상테어서 캐릭터를 움직이면 조금 어질어질하기도 하네요.

**Comment by shlee303** [comment]
현재는 의도된 기능인데, 고정이 사용자 입장에서 정상적인 동작으로 가늠된다면 고정 될 수 있게 변경하는 방안을 고민해보겠습니다! 감사합니다!

**Comment by shlee303** [comment]
@DolDolDol 해당 부분 기능으로 반영 완료 했습니다. 이제 우클릭하여 좌/우/위/아래 고정 됩니다. 좋은 의견 정말 감사합니다~
